src/test/ui/issues/issue-77218.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
```

Comment:
What is this test for?

Should probably be moved to a more specific directory than `issues`

Reply:
It's a regression test for a previous ICE in #77218. The code is taken verbatim from [this comment](https://github.com/rust-lang/rust/issues/77218#issuecomment-702107741). Where should it be placed? The issues directory is typically used for ICE regression tests, no?

Reply:
Wait, I'm confused...this test [already exists](https://github.com/rust-lang/rust/blob/master/src/test/ui/issues/issue-77218/issue-77218.rs) and the associated issue is already closed? Why is this getting added?

Reply:
Whoops. Just looked at the full diff and see that this test already exists. I probably just messed something up with tab autocomplete on the command line and never noticed.

I'll get this fixed tomorrow most likely and approve it on your review at that point (I think I'm able to do that, not positive).